curriculum/challenges/english/21-a2-english-for-developers/learn-how-to-ask-and-share-about-educational-and-professional-background/657b135e9029fb4f8141e40c.md

```diff
@@ -0,0 +1,48 @@
+---
+id: 657b135e9029fb4f8141e40c
+title: Task 14
+challengeType: 22
+dashedName: task-14
+audioPath: curriculum/js-music-player/We-Are-Going-to-Make-it.mp3
+---
+
+<!--
+AUDIO REFERENCE:
+Anna: That's excellent. Were there any specific projects you'd like to mention?
+Candidate: Yes, in one of my team's projects, we were involved in the development of an e-commerce platform. We were responsible for the checkout process, and it was a successful implementation.
+-->
+
+# --description--
+
+Listen to the dialogue and fill in the blanks with the correct words. This is a great way to practice and remember these words.
+
+# --fillInTheBlank--
+
+## --sentence--
+
+`Anna: That's excellent. _ there any specific projects you'd like to mention?`
+`Candidate: Yes, in one of my team's projects, we were _ in the development of an e-commerce platform. We were responsible for the _ process, and it was a successful implementation.`
+
```

Comment:
If this sentence with multiple characters doesn't look good we can Delete Anna's line and focus on the candidate.

Reply:
```suggestion
`Anna: That's excellent. _ there any specific projects you'd like to mention?`

`Candidate: Yes, in one of my team's projects, we were _ in the development of an e-commerce platform. We were responsible for the _ process, and it was a successful implementation.`

```
If https://github.com/freeCodeCamp/freeCodeCamp/pull/52552 goes in, this is the correct formatting. Without it it doesn't make much difference.